app/dashboard/models.py

```diff
@@ -74,27 +99,38 @@ class Meta:
     bounty_owner_address = models.CharField(max_length=50)
     bounty_owner_email = models.CharField(max_length=255, null=True)
     bounty_owner_github_username = models.CharField(max_length=255, null=True)
+    bounty_owner_name = models.CharField(max_length=255, null=True)
     claimeee_address = models.CharField(max_length=50)
-    claimee_email = models.CharField(max_length=255, null=True)
-    claimee_github_username = models.CharField(max_length=255, null=True)
-    is_open = models.BooleanField(help_text='Whether the bounty has been filled')
+    claimee_email = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
@owocki @jasonrhaas  Can we adjust all of these text/char fields to: `blank=True` vs `null=True`?  Per https://docs.djangoproject.com/en/2.0/ref/models/fields/#null

Reply:
@mbeacom Sounds good to me.

Reply:
👍